src/main/scala/net/codingwell/scalaguice/InjectorExtensions.scala

```diff
@@ -25,5 +25,7 @@ object InjectorExtensions {
     def instance[T: Manifest] = i.getInstance(typeLiteral[T].toKey)
     def instance[T: Manifest](ann: Annotation) = i.getInstance(typeLiteral[T].annotatedWith(ann))
     def instance[T: Manifest, Ann <: Annotation : Manifest] = i.getInstance(typeLiteral[T].annotatedWith[Ann])
+
+    def existingBinding[T: Manifest] = Option(i.getExistingBinding(typeLiteral[T].toKey))
```

Comment:
Please add the annotation variants as well. (i.e. the other variants of the 'instance' methods)